config/app.default.php

```diff
@@ -9,7 +9,7 @@
      * Development Mode:
      * true: Errors and warnings shown.
      */
-    'debug' => true,
+    'debug' => env('DEBUG', true),
```

Comment:
`0` or `1` are pretty common alternative values  for booleans, and PHP treats them more or less fine (in debug's case, I believe it is completely fine.).